lib/deploy.js

```diff
@@ -8,7 +8,7 @@ const tgzCompress = promisify(compress);
 
 const { getAPIKey } = require('./userConf');
 const { deploy } = require('../sdk');
-const { invokeEndpoint } = require('../sdk/config');
+const { getInvokeEndpoint } = require('../sdk/config');
```

Comment:
Once we release this as an SDK, any such change will require a major version release.  Let's decide on a naming convention?

Reply:
I'll respond to both of your comments here.

1. If you notice previously the environment variables were loaded as static variables directly into the relevant SDK files. This meant that even if the ENV's changed during runtime it wouldn't reflect in the `required` code because the ENV's had been cached (so to speak). Now that an actual function is used the ENV's are pulled dynamically.

2. Agree and we had a small discussion on slack about this. Is an issue the best place or will another medium work better?